### PR TITLE
chore: use gts

### PR DESCRIPTION
**Pull request by fhinkel**
Fix typo in .eslintrc.json (had an extra t before).
    
Fix one usage of an invalid export. 

Convert `it.only` to `it.skip` for failing tests in iot, see https://github.com/GoogleCloudPlatform/nodejs-docs-samples/issues/2026
    
Disable a bunch of rules to get this to pass. We'll need to turn them on
and fix their usages manually. Exclude `run/idp-sql/` for now from linting, too many errors.

### CODE REVIEW

**Review by grant — DISMISSED** [review]
It's generally recommended to use parentheses even for single-parameter arrow functions.

https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions

Otherwise looks fine for `functions/`

**Review by kurtisvg — APPROVED** [review]
LGTM for `/cloudsql`

**Review by BenWhitehead — APPROVED** [review]
LGTM for `datastore/`

**Review by leahecole — APPROVED** [review]
lgtm for composer

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "./node_modules/gts/",
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+    "node/no-missing-require": ["off"], 
+    "node/no-unpublished-require": ["off"], 
+    "no-unused-vars": ["off"],
```
Comment (ace-n) [inline .eslintrc.json:9]: Ditto - why is this disabled?

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "./node_modules/gts/",
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+    "node/no-missing-require": ["off"],
```
Comment (ace-n) [inline .eslintrc.json:7]: What's the reason for disabling this?

**Review by ace-n — CHANGES_REQUESTED** [review]

**Review by ricalo — APPROVED** [review]
LGTM for speech-to-speech/

**Comment by JustinBeckwith** [comment]
@grant on the parens rule - I'm onboard with changing it, but we should do it upstream in `gts` as a default.  Feel free to enable it here broadly if you like, but we should try to follow the google style guide best as we can over in gts.

**Review by JustinBeckwith — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -13,8 +13,8 @@
   },
   "private": true,
   "scripts": {
-    "lint": "eslint '**/*.js'",
```
Comment (JustinBeckwith) [inline package.json:16]: Can we also remove the direct dependencies on `eslint`, `prettier`, and any other `eslint` plugins we're using and now don't need?

File: .eslintrc.json (hunk 1 above)
Comment (ace-n) [reply .eslintrc.json:9]: Ah NVM, saw your comment above - but can you file a GitHub issue for tracking?

File: .eslintrc.json (hunk 2 above)
Comment (ace-n) [reply .eslintrc.json:7]: @fhinkel mind adding this to said (tracking) GitHub issue too?

**Comment by grant** [comment]
> @grant on the parens rule - I'm onboard with changing it, but we should do it upstream in `gts` as a default.  Feel free to enable it here broadly if you like, but we should try to follow the google style guide best as we can over in gts.

Upstream in gts sounds good 👍

I don't think we'd want all of the function samples to show up in the non-paren style and then switch back at some point later, so maybe we can change the PR to add a rule or first change gts?

File: .eslintrc.json (hunk 1 above)
Comment (fhinkel) [reply .eslintrc.json:9]: See PR description

File: .eslintrc.json (hunk 2 above)
Comment (fhinkel) [reply .eslintrc.json:7]: See PR description

File: package.json (hunk 3 above)
Comment (fhinkel) [reply package.json:16]: good catch, thanks.

**Comment by fhinkel** [comment]
We'd be rolling out this change in 60+ libraries. I think it's better to have consistency among our samples than to discuss a particular style.

**Review by ace-n — APPROVED** [review]

**Pull request merged**